tests/unittest.py

```diff
@@ -123,7 +123,11 @@ def create_peer(self, network, peer_id=None, wallet=None, tx_storage=None, unloc
             if unlock_wallet:
                 wallet.unlock(b'MYPASS')
         if tx_storage is None:
-            tx_storage = TransactionMemoryStorage()
```

Comment:
Our tests use `TransactionMemoryStorage()` by default. So we are not testing our in-disk indexes for most of the tests. Should we change it somehow?

Reply:
Maybe the default can be configured though settings, or at least have some env var that the pytest setup will load to choose one or the other. RocksDB storage isn't available on Windows, it would be useful to make it configurable on the CI.

Reply:
Is that the reason the CI is not ok for windows?

Reply:
Yes.

Reply:
This is fixed.